aws-single-page-static-site/main.tf

```diff
@@ -87,7 +87,7 @@ resource "aws_s3_bucket_public_access_block" "bucket" {
 module "security_headers_lambda" {
   source = "../aws-lambda-edge-add-security-headers"
 
-  function_name = replace("${local.website_fqdn}-static-site-security-headers", ".", "-")
+  function_name = replace("${local.website_fqdn}-headers", ".", "-")
```

Comment:
Reduce issue where this string is >64 characters long. Forces redeploy of lambda for anything already deployed with HSTS headers (which should just be a single deployment in Along), and the redeploy should be safe.